docs/cellxgene_census_schema.md

```diff
@@ -872,8 +881,185 @@ Cell metadata MUST be encoded as a `SOMADataFrame` with the following columns:
+
+#### Matrix Data, count (raw) matrix – `census_obj["census_spatial_sequencing"][organism].ms["RNA"].X["raw"]` – `SOMASparseNDArray`
+
+Same as non-spatial data. See the corresponding section [here](#matrix-data-count-raw-matrix--census_objcensus_dataorganismmsrnaxraw--somasparsendarray).
+
+#### Feature metadata – `census_obj["census_spatial_sequencing"][organism].ms["RNA"].var` – `SOMADataFrame`
+
+Same as non-spatial data. See the corresponding section [here](#feature-metadata--census_objcensus_dataorganismmsrnavar--somadataframe).
+
+#### Feature dataset presence matrix – `census_obj["census_spatial_sequencing"][organism].ms["RNA"]["feature_dataset_presence_matrix"]` – `SOMASparseNDArray`
+
+Same as non-spatial data. See the corresponding section [here](#feature-dataset-presence-matrix--census_objcensus_dataorganismmsrnafeature_dataset_presence_matrix--somasparsendarray).
+
+#### Cell metadata – `census_obj["census_spatial_sequencing"][organism].obs` – `SOMADataFrame`
+
+Same as non-spatial data. See the corresponding section [here](#cell-metadata--census_objcensus_dataorganismobs--somadataframe).
+
+**Important note:** In addition, the following spatial `obs` columns from the CELLxGENE dataset schema MUST be included in this `SOMADataFrame`
+
+<table>
+<thead>
+  <tr>
+    <th>Column</th>
+    <th>Encoding</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>array_col</td>
+    <td colspan="2" rowspan="3">As defined in CELLxGENE dataset schema</td>
+  </tr>
+  <tr>
+    <td>array_row</td>
+  </tr>
+  <tr>
+    <td>in_tissue</td>
+  </tr>
+</tbody>
+</table>
+
+#### Obs to spatial mapping –  `census_obj["census_spatial_sequencing"][organism].obs_presence_matrix` – `SOMADataFrame`
+
+It indicates the link between an observation and a scene.  Each row corresponds to an observation with the following columns:
+
+<!-- markdownlint-disable reference-links-images -->
+<table>
+<thead>
+  <tr>
+    <th>Column</th>
+    <th>Encoding</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>soma_joinid</td>
+    <td>int</td>
+    <td>It MUST be valid <code>soma_joinid</code> from <code>census_obj["census_spatial_sequencing"][organism].obs</code>.</td>
+  </tr>
+  </tr>
+    <td>scene_id</td>
+    <td>string</td>
+    <td>It MUST be valid <code>scene_id</code> from <code>census_obj["census_spatial_sequencing"][organism].spatial</code>.</td>
+  </tr>
+  <tr>
+    <td>data</td>
+    <td>bool</td>
+    <td>It MUST be <code>True</code> if the scene contains spatial information about the oberservation, otherwise it MUST be <code>False</code>.</td>
+  </tr>
+  </tbody>
+</table>
+<!-- markdownlint-enable reference-links-images -->
+
+#### Positions array of a Scene – `census_obj["census_spatial_sequencing"][organism].spatial[scene_id].obsl["loc"]` – `PointCloudDataFrame`
```

Comment:
RE `PointCloudDataFrame` - should this be `SOMAPointCloudDataFrame`

Reply:
Ah, looks like it's from the SOMA spec

Reply:
That's where I discovered it. May be helpful to have links to the SOMA specification for readers/reviewers where appropriate.